src/ui/pages/SwapForm/fieldsets/FiatInputValue/FiatInputValue.tsx

```diff
@@ -41,40 +48,59 @@ export function FiatInputValue({
     asset?.price?.value || 0
   );
 
-  let diff: BigNumber | null = null;
-  if (name === 'receiveInput') {
-    const oppositeFiatValue = new BigNumber(oppositeInputValue || 0).times(
-      oppositeAsset?.price?.value || 0
-    );
-    diff = oppositeFiatValue.isGreaterThan(0)
-      ? fiatValue.minus(oppositeFiatValue).div(oppositeFiatValue)
-      : null;
-  }
+  return (
+    <UIText kind="small/regular" color={color} style={style} title={title}>
+      {isPrimaryInput ? null : '≈'}
+      {formatCurrencyValue(fiatValue, 'en', currency)} {percentageChangeText}
+    </UIText>
+  );
+}
 
-  const formattedDiff = diff
-    ? `${diff.isLessThan(0) ? '' : '+'}${formatPercent(diff.times(100), 'en')}%`
-    : null;
+export function SpendFiatInputValue({ swapView }: { swapView: SwapFormView }) {
```

Comment:
Do we need this wrapper?

Reply:
Just though it reads better, I mean having `SpendFiatInputValue` / `ReceiveFiatInputValue` instead of just `FiatIputValue` and `ReceiveFiatInputValue`